pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig_test.go

```diff
@@ -94,6 +94,23 @@ var _ = Describe("Machine Config", func() {
 		})
 	})
 
+	Context("machine config creation with disabled RPS", func() {
+		It("should create machine config with no trace of RPS", func() {
+			profile := testutils.NewPerformanceProfile("test")
+			profile.Annotations = map[string]string{}
+			profile.Annotations[performancev2.PerformanceProfileEnableRpsAnnotation] = "false"
+
+			mc, err := New(profile, &components.MachineConfigOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			y, err := yaml.Marshal(mc)
+			Expect(err).ToNot(HaveOccurred())
+
+			manifest := string(y)
+			Expect(manifest).To(Not(ContainSubstring("rps")))
```

Comment:
Usually the content is base64 coded.
I think we should decode the content first and than perform this check.

Reply:
Well.. true it could happen. I do not think it is likely (especially in the filename). This allows a check without parsing the text back to yaml. A crude one, yes, but it works and catches both the systemd unit and the sysctl file.

Reply:
>This allows a check without parsing the text back to yaml.

Which is not so bad especially since we're talking about tests where we prefer correctness over performance.